libs/blocks/quiz-entry/mlField.js

```diff
@@ -31,6 +31,21 @@ export const getFiResults = async (endpoint, apiKey, input, numberOfItems, valid
   return result;
 };
 
+const handleInput = () => {
```

Comment:
Ideally, we could just update the state of all of the cards and add or removed disabled from them, allowing the standard card machinery to take effect, rather than branching it to add classes as we are here. I'm not sure how to do this, but if we could pull that off, we wouldn't have two ways of disabling, which is confusing. Any ideas on how we can just update card state from the field @fullcolorcoder ?

One reason that this is problematic, is that we are missing the bit that adds the disabled attribute, which is what actually prevents tabbing on the card. If you enter text in the field, and tab through, it should go straight to the continue button, skipping the cards. However, since the cards are only getting a class, they look disabled, but aren't actually disabled.